compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/PlatformClipboard.desktop.kt

```diff
@@ -44,8 +45,8 @@ internal class AwtPlatformClipboard internal constructor() : Clipboard {
     override suspend fun setClipEntry(clipEntry: ClipEntry?) {
         val transferable = clipEntry?.nativeClipEntry as? Transferable
         systemClipboard?.setContents(
-            transferable ?: EmptyTransferable,
-            null
+            /* contents = */ transferable ?: EmptyTransferable,
+            /* owner = */ transferable as? ClipboardOwner,
```

Comment:
I think this was technically wrong before, as we are supposed to provide the owner to the AWT clipboard if there is one. Given the `Transferable` itself is the only way we could have an owner, if it implements `ClipboardOwner` we pass it as owner.

`ClipboardOwner` is a simple interface that only provides a callback to know "you're not the owner of the clipboard any longer", meaning something else was put in the clipboard. I think this can be needed to do some cleanup in some `Transferable`s which may be holding on to resources internally.